deeposlandia/datagen.py

```diff
@@ -0,0 +1,118 @@
+"""Main method to generate new datasets
+"""
```

Comment:
Maybe you can add an usage example of this script, e.g.

```to generate data about the xxx dataset, do

python datagen.py --args ...
```

Reply:
Two examples have been added to the module docstring.